### PR TITLE
Fix style around example codes

### PR DESCRIPTION
**Pull request by yannicka**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | Apply some style corrections around the code examples. And correct some typos.
| Fixed ticket? | N/A

### CODE REVIEW

File: basics/keeping-up-to-date/backup.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ tar -czf <file_name>.tar <folder_to_save>
 ```
 
 For instance:
+
```
Comment (matks) [inline basics/keeping-up-to-date/backup.md:44]: Hello, can you explain why adding these white spaces is a good thing? I am not sure it provides value 🤔

Comment (yannicka) [reply basics/keeping-up-to-date/backup.md:44]: This is simply to separate each block/paragraph visually. I can undo this if I need to.

**Review by kpodemski — APPROVED** [review]
Good for me 👍 Let's wait for @thomasnares opinion too :)

**Comment by thomasnares** [comment]
Yes, easier to maintain with one white space between content and code when working on .md 👍

**Review by thomasnares — APPROVED** [review]

**Comment by kpodemski** [comment]
Thank you @yannicka

**Pull request merged**